apps/server/Makefile

```diff
@@ -40,6 +40,12 @@ SRC = main.c \
 BACNET_BASIC_SRC += \
 	$(wildcard $(BACNET_SRC_DIR)/bacnet/basic/service/*.c)
 
+# build in uci integration - use UCI=1 when invoking make
```

Comment:
This should be moved to the new (old lib) apps/lib/Makefile

Reply:
OK